README.md

```diff
@@ -291,6 +291,25 @@ try {
 }
 ```
 
+To enable caching for the requests, one can add `expiresIn` to the requests configuration object. The values are in seconds.
+```
+// cache is valid for 30 minutes
+const requestsConfig = {
+  cache: {
+  expiresIn: 1800,
```

Comment:
Typo: indentation is wrong.

EDIT: below also.

Reply:
Fixed